lib/passfort.rb

```diff
@@ -8,6 +8,21 @@
 require "passfort/client"
 
 module Passfort
+  module CheckType
+    IDENTITY_CHECK = "IDENTITY_CHECK"
+    DOCUMENT_VERIFICATION = "DOCUMENT_VERIFICATION"
+    DOCUMENT_FETCH = "DOCUMENT_FETCH"
+    PEPS_SCREEN = "PEPS_SCREEN"
+    SANCTIONS_SCREEN = "SANCTIONS_SCREEN"
+    PEPS_AND_SANCTIONS_SCREEN = "PEPS_AND_SANCTIONS_SCREEN"
+    ADVERSE_MEDIA_SCREEN = "ADVERSE_MEDIA_SCREEN"
```

Comment:
Couldn't find this check type in the docs either

Reply:
This one appears in their check types list, but it is not being used. I'll remove it